public/js/guests/controllers/guestListController.js

```diff
@@ -34,14 +34,9 @@ define([
             var alerts = validateGuest(newGuest);
 
             if (newGuest && alerts.length === 0) {
-                $http.post("/api/v1/guests", newGuest)
-                    .error(function (data) {
-                        $scope.alerts.push({msg: data, type: "danger"});
-                    })
-                    .success(function (data) {
-                        newGuest._id = data.id;
-                        $scope.guests.push(newGuest);
-                    });
+                GuestServices.save(newGuest, function() {
+                    $scope.guests.push(newGuest);
+                });
```

Comment:
Hook into the error callback to add the error message. I've needed this before when the mongo DB in the cloud behaved slightly differently with id's than the local one.